packages/data/README.md

```diff
@@ -470,7 +474,8 @@ _Returns_
 
 ### dispatch
 
-Given the name of a registered store, returns an object of the store's action creators.
+Given a store descriptor (also supports legacy calling convention accepting the name of a
```

Comment:
The term "store descriptor" here isn't really clear to me.

Reply:
Hey @ZebulanStanphill! Thank you for checking it!
I added an explanation of what is the store descriptor in the param. WDYT?

https://github.com/WordPress/gutenberg/pull/41210/commits/973c18a4245bf42e37a7343e20b62b7b3f54fa25

Reply:
Yeah, that makes it clearer. :+1:

Reply:
this probably isn't the best place to define "store descriptor" though the confusion is understandable. we've recently broadened the use of the term in order to eliminate ambiguity with different names.

in order to prevent bloating these docblock areas and to prevent having very inconsistent explanation of the `descriptor` name, maybe we can find a place to define it in a core data glossary.

either way, I'd rather not introduce a more verbose description here because it does open a question of how far we go and how much we duplicate what really ends up being a fundamental term in the data system